CorsixTH/Lua/map.lua

```diff
@@ -207,22 +201,19 @@ function Map:load(level, difficulty, level_name, map_file, level_intro, map_edit
       local demo_path = self.app:getFullPath({"Levels", "demo.level"})
       errors, result = self:loadMapConfig(demo_path, base_config, true)
       if errors then
-        print("Warning: Could not find the demo configuration, try reinstalling the game")
+        return nil, _S.errors.missing_corsixth_file:format("demo.level")
       end
+      self.difficulty = "full"
       self.level_config = result
     else
-      local level_no = level
-      if level_no < 10 then
-        level_no = "0" .. level
+      errors, result = self:_loadOriginalCampaignLevel(difficulty, level, base_config)
+      if errors then
+        return nil, errors
       end
-      -- Override with the specific configuration for this level
-      _, result = self:loadMapConfig(difficulty .. level_no .. ".SAM", base_config)
-      -- Finally load additional CorsixTH config per level
-      local level_path = self.app:getFullPath({"Levels", "original" .. level_no .. ".level"})
-      _, result = self:loadMapConfig(level_path, result, true)
       self.level_config = result
     end
   elseif map_editor then
+    local _
```

Comment:
Do we usually have to declare the underscore for an unwanted value?
I suppose it could potentially leak into the global scope otherwise.

Reply:
Yes we have to